src/subscription.ts

```diff
@@ -131,13 +131,13 @@ export function getSubscriptionPlan(
 export function getSubscriptionPlanName(id: SubscriptionPlanId) {
 	switch (id) {
 		case SubscriptionPlanId.FreePlus:
-			return 'GitLens Free';
+			return 'GitKraken Free';
 		case SubscriptionPlanId.Pro:
-			return 'GitLens Pro';
+			return 'GitKraken Pro';
 		case SubscriptionPlanId.Teams:
-			return 'GitLens Teams';
+			return 'GitKraken Teams';
 		case SubscriptionPlanId.Enterprise:
-			return 'GitLens Enterprise';
+			return 'GitKraken Enterprise';
 		case SubscriptionPlanId.Free:
 		default:
 			return 'GitLens';
```

Comment:
Should this be 'GitKraken'? This would be the license displayed by your name in the GitKraken Account view when you're on the "Free" plan.

Reply:
Ok, I'll update it, thanks!